sdk/storage/storage-common/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0 (Unreleased)
+
```

Comment:
storage-common is not a package, seems we do not need a separate changelog.md and license.

Reply:
Sounds good.